src/js/api/errors.ts

```diff
@@ -72,6 +72,11 @@ export const ASYNC_PLAYLIST_ITEM_REJECTED = 203700;
  */
 export const ERROR_LOADING_PROVIDER = 204000;
 
+/**
+ * @enum {ErrorCode} The LiveSyncDuration is higher than 45
+ */
+ export const SETUP_ERROR_LIVESYNCDURATION = 300100;
```

Comment:
NOTE: I decided that 3001xx would be reserved for future config validation messages. We can always use 3002xx (or higher) if need be since the first defined error after this is 301xxx space. I left 3000xx free just in case...